app/phone/src/main/java/dev/jdtech/jellyfin/PlayerActivity.kt

```diff
@@ -342,6 +347,23 @@ class PlayerActivity : BasePlayerActivity() {
         } catch (_: IllegalArgumentException) { }
     }
 
+    private fun showQualitySelectionDialog() {
+       val height = viewModel.getOriginalHeight() // TODO: rewrite getting height stuff I don't like that its only update after changing quality
+        val qualities = when (height) {
+            0 -> arrayOf("Auto", "Original - Max", "720p - 2Mbps", "480p - 1Mbps", "360p - 800kbps")
```

Comment:
This cannot be localized because the strings are hardcoded. You should use `strings.xml` for this so the strings can be translated in weblate